src/components/widget/List/RawList.js

```diff
@@ -370,6 +378,13 @@ export class RawList extends PureComponent {
       value = selected.caption;
     }
 
+    // set custom placeholder based on field ID
+    const { field, caption } = this.props.properties;
+    placeholder = field === C_BPartner_Location_ID ? caption : placeholder;
```

Comment:
So I think instead of hardcoding the field names here, we should make this work for all combined lookups. So just grab `lookupList` from the props and whenever it's true, use the caption. Otherwise - placeholder.

![Screen Shot 2020-01-08 at 11 58 00](https://user-images.githubusercontent.com/513460/71974528-fe452a80-3211-11ea-8bd0-e942c6111d79.png)

Reply:
Those are not hardcoded. There is a map file for each of the fields IDs. I'll try your approach as well